ui/app/src/components/TitleCard.tsx

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Keep this components together with the GlobalNav file until we see use for there some other place. 

Rename to SiteCard or find a better name.